Utest/rimea_tests/test_4/runtest_rimea_4.py

```diff
@@ -54,13 +54,15 @@
 sys.path.append(utestdir)
 from JPSRunTest import JPSRunTestDriver
 from utils import *
+import time
 
-def run_rimea_test4():
+def run_rimea_test4(inifile, trajfile):
     # Catch all Traji-files
     files = glob.glob("trajectories/*")
     
-    density = [2.0,3.0,4.0,0.5,1.0,1.5] #DO NOT CHANGE THE ORDER
-    
+#    density = [2.0,3.0,4.0,0.5,1.0,1.5] #DO NOT CHANGE THE ORDER
+    density =  [2.0,1.0,6.0,3.0,4.0,5.0,0.5]
```

Comment:
I think should be "density=[2, 3, 4, 0.5, 5, 6, 1]", according to the name of trajecory files

Reply:
> I think should be "density=[2, 3, 4, 0.5, 5, 6, 1]", according to the name of trajecory files
The order in windows is like this.May be you can sort trajectories to avoid it.

![image](https://user-images.githubusercontent.com/32732507/62083632-b3cd8400-b257-11e9-8dce-4a5e8f1ba56e.png)

Reply:
@GuidoBasten Why do not you sort in ascending order?

Reply:
@GuidoBasten this is actually a good question.